packages/helper/src/main/ts/utils/withRateLimit.ts

```diff
@@ -0,0 +1,52 @@
+import { DeepProxy } from '@qiwi/deep-proxy'
+import { ratelimit } from 'push-it-to-the-limit'
+import { IControlled } from 'push-it-to-the-limit/target/es5/interface'
+
+import { TRateLimitOpts } from '../helper'
+import { convertRateLimitOpts } from './converters'
+
+export const applyMemoization = <R>(
```

Comment:
This would be very unexpected behaviour for long-term sessions using this client. If it's necessary, 1 minute or less (this param should be configurable by opts). Use @qiwi/primitive-storage